group/assignment6/assignment6.py

```diff
@@ -8,44 +8,7 @@
 Car #1 is in the lot #0, car #2 is in the lot #1, etc.
 """
 
-
-def move_to_empty_lot(current_state, inverted_state, move_from, move_to):
-    """
-    Move a car from one slot to another. As we are always moving a car to
-    an empty slot, the function basically swaps an empty and occupied slots
-    in the current state and corresponding lots in the inverted state
-
-    :param current_state: order of cars in the present moment
-    :param move_from: int, number of a lot from which the car is moved
-    :param move_to: int, number of an empty slot to which we move the car
-    :returns: modified current state
-    """
-    current_state[move_from], current_state[move_to] = current_state[move_to], \
-                                                       current_state[move_from]
-    inverted_state[current_state[move_from]], inverted_state[current_state[move_to]] =\
-    inverted_state[current_state[move_to]], inverted_state[current_state[move_from]]
-    return current_state, inverted_state
-
-
-def compute_inverted_state(state):
-    inverted_state = list(range(len(state)))
-    for lot, car in enumerate(state):
-        inverted_state[car] = lot
-    return inverted_state
-
-
-def find_empty(inverted_state):
-    """
-    :returns: int, number of a currently empty lot
-    """
-    return inverted_state[0]
-
-
-def find_car_lot(inverted_state, car):
-    """
-    :returns: int, number of a lot in which a given car is currently placed
-    """
-    return inverted_state[car]
+from parking_class import Parking
```

Comment:
I don't know the right way of importing in python, now your implementation doesn't work for me because my directory is assignment6, so for me the row  assignment6.parking_class import Parking will work, does it work for you too?

Reply:
for me it works the way i wrote. if i change to assignment6.some_name it won't work. i run python from console so it looks for the package in the current directory.